src/bucketblocker/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/sts"
+)
+
+func blockPublicAccess(name string, s3Client s3.S3) (*s3.PutPublicAccessBlockOutput, error) {
```

Comment:
Generally speaking I think big structures (such as clients) should be passed by ref rather than value, i.e. `s3Client *s3.S3`. 
Also a personal pet peeve, but I think the client should be the first argument